### PR TITLE
Generate the default infoPlist file for resources targets that only contain resources

### PR DESCRIPTION
**Pull request by Juanpe**
Resolves https://github.com/tuist/tuist/issues/1602

### Short description 📝

As @Mateusz-Chechlinski said, there is a problem with static frameworks that only contain resources. In this scenario, **tuist** doesn't generate an `Info.plist` file for this target, so it's not possible to use any resources that belong to this static framework.

### Solution 📦

To solve this problem we need to set a default `Info.plist` while mapping this resources target. 
Also, to be sure that the info plists mapper takes place after the resources mapper we need to swap the order in the mappers pipeline.

### Implementation 👩‍💻👨‍💻
- [x] Set `infoPlist: .extendingDefault(with: [:])` on the Resource target in the [ResourceMapper](https://github.com/tuist/tuist/blob/master/Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift#L32).
- [x] Swap the mapper's pipeline order to ensure the info plists mapper takes place after the resources mapper.
- [x] Update fixtures to validate this scenario.

### CODE REVIEW

**Comment by Juanpe** [comment]
Thanks! Glad to contribute 🙂

File: fixtures/ios_app_with_framework_and_resources/App/Sources/AppDelegate.swift [hunk 1]
```diff
@@ -19,10 +20,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
         print("Main bundle image: \(String(describing: UIImage(named: "tuist")))")
-        print("StaticFrameworkResouce image: \(String(describing: staticFrameworkResources.tuist))")
-        print("StaticFramework2Resouce image: \(String(describing: staticFramework2Resources.loadImage()))")
-        print("StaticFramework3Resouce image: \(String(describing: resourcesStaticFramework3.loadImage()))")
-        print("StaticFramework4Resouce image: \(String(describing: resourcesStaticFramework4.loadImage()))")
+        print("StaticFrameworkResource image: \(String(describing: staticFrameworkResources.tuist))")
+        print("StaticFramework2Resource image: \(String(describing: staticFramework2Resources.loadImage()))")
+        print("StaticFramework3Resource image: \(String(describing: resourcesStaticFramework3.loadImage()))")
+        print("StaticFramework4Resource image: \(String(describing: resourcesStaticFramework4.loadImage()))")
+        print("StaticFramework5Resource image: \(String(describing: UIImage(named:"StaticFramework5Resources-tuist", in: StaticFramework5Resources.bundle, compatibleWith: nil)))")
```
Comment (kwridan) [inline fixtures/ios_app_with_framework_and_resources/App/Sources/AppDelegate.swift:27]: minor: For this fixture to help represent the issue in https://github.com/tuist/tuist/issues/1602, `StaticFramework5` would need an asset catalog with the image within it.

Comment (Juanpe) [reply fixtures/ios_app_with_framework_and_resources/App/Sources/AppDelegate.swift:27]: Hi @kwridan, I did the change, but I've refreshed my branch with `master` and now, the project doesn't compile :/ Is there any important change or something that I should do?

Comment (Juanpe) [reply fixtures/ios_app_with_framework_and_resources/App/Sources/AppDelegate.swift:27]: Forget my comment, it's solved 👍 
I've added an asset catalog with the image inside

Comment (kwridan) [reply fixtures/ios_app_with_framework_and_resources/App/Sources/AppDelegate.swift:27]: ouch, what error are you seeing?

Comment (Juanpe) [reply fixtures/ios_app_with_framework_and_resources/App/Sources/AppDelegate.swift:27]: no problem, it was a problem related to my IDE :) 
I've updated the PR with the changes 👍

**Pull request merged**